hotsos/defs/scenarios/storage/nfs/nfs-detect-mount-name-resolution-failure.yaml

```diff
@@ -0,0 +1,16 @@
+checks:
+  has_nfs_name_resolution_failures:
+    input:
+      path: ['var/log/syslog', 'var/log/boot.log']
+    expr:
+      - '.*mount.nfs.*Failed to resolve server (.*): (Name or service not known|Temporary failure in name resolution).*'
+conclusions:
+  nfs_name_resolution_failures:
+    decision: has_nfs_name_resolution_failures
+    raises:
+      type: NFSNameResolutionError
+      message: >-
+        The following name resolution failures detected for the NFS mounts:
```

Comment:
Name resolution failures have been detected for the following NFS mounts:

Reply:
fixed